libs/test/Test/Golden.idr

```diff
@@ -246,15 +247,15 @@ runTest opts testPath = forkIO $ do
         Nothing => putStr $ unlines
           [ "Golden value missing. I computed the following result:"
           , out
-          , "Accept new golden value? [yn]"
+          , "Accept new golden value? [y/N]"
```

Comment:
If the prompt is changed to uppercase it might be good to change L234 to `case toLower str of` to avoid casing confusion.